lib/active_interaction/extras/strong_params.rb

```diff
@@ -5,6 +5,8 @@ def initialize(inputs = {})
     # TODO: whitelist :params and :form_params, so they could not be used as filters
     return super if self.class.filters.key?(:params) || self.class.filters.key?(:form_params)
 
+    return super(inputs) unless %i[key? fetch merge].all? { |m| inputs.respond_to? m }
```

Comment:
Do we need to explicitly pass `inputs` to `super` here?

Reply:
Nope, `super` on its own should suffice. The `super` call below on line 27 passes `inputs` explicitly and I followed that. I suppose we could probably combine this conditional early return with the one above. WDYT?
```
    return super if 
      self.class.filters.key?(:params) ||
      self.class.filters.key?(:form_params) ||
      !%i[key? fetch merge].all? { |m| inputs.respond_to? m }
```

Reply:
Seems good to me!